remove_employee_webhook_receiver.py

```diff
@@ -0,0 +1,52 @@
+import traceback
+import json
+import requests
+
+from ssm_util import fetch_ssm_params
+
+# Removes an employee from the guide
```

Comment:
nitpick: I'd prefer this as a docstring to the `lambda_handler()` function. On that note, you can give the lambda handler function here and in the other lambda functions a more descriptive name, like `remove_employee_from_guide()` and `add_employee_to_guide()`